WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/Payments/StoreOnboardingPaymentsSetupCoordinator.swift

```diff
@@ -78,11 +78,10 @@ private extension StoreOnboardingPaymentsSetupCoordinator {
         }
 
         let webViewModel = WooPaymentSetupWebViewModel(title: title, initialURL: url) { [weak self] success in
+            self?.dismissWebview()
```

Comment:
Should we wait for the completion of the dismissal for extra safety?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10594/commits/6d65ada8a92396267d5a4ba06c564bdfc5d2e0d2